context-linux/src/usr/bin/cloudinit_cc_run_cmd.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
+
+# -------------------------------------------------------------------------- #
+# Copyright 2002-2024, OpenNebula Project, OpenNebula Systems                #
+#                                                                            #
+# Licensed under the Apache License, Version 2.0 (the "License"); you may    #
+# not use this file except in compliance with the License. You may obtain    #
+# a copy of the License at                                                   #
+#                                                                            #
+# http://www.apache.org/licenses/LICENSE-2.0                                 #
+#                                                                            #
+# Unless required by applicable law or agreed to in writing, software        #
+# distributed under the License is distributed on an "AS IS" BASIS,          #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   #
+# See the License for the specific language governing permissions and        #
+# limitations under the License.                                             #
+#--------------------------------------------------------------------------- #
+
+require 'fileutils'
+
+module CloudInit
+
+    ##
+    # RunCmd class implements the runcmd cloud-config directive.
+    ##
+    class RunCmd
+
+        attr_accessor :cmd_list
+
+        def initialize(cmd_list)
+            unless cmd_list.is_a?(Array)
```

Comment:
Please consider converting constructions like this one (an if-block with a single instruction inside) to something like:
```ruby
raise 'RunCmd must be instantiated with a command list as an argument' \
    unless cmd_list.is_a?(Array)
```
that's what for example rubocop always complains about. :face_holding_back_tears:

Reply:
Yeah, I made it because rubocop was complaining about a long line, but I just discovered that I can use a backslash for splitting lines, like in bash scripts